example/dev.tfvars

```diff
@@ -0,0 +1,3 @@
+bucket_name              = "arc-mayank-poc"
```

Comment:
Change to something generic please

Reply:
sorry i just was in a hurry to shutdown :sweat_smile:
Done